weasyprint/css/properties.py

```diff
@@ -97,6 +97,8 @@
     'quotes': list('“”‘’'),  # depends on user agent
     'position': 'static',
     'right': 'auto',
+    #default not defined in spec http://dev.w3.org/csswg/css-gcpm/#named-strings
+    'string_set': '',
```

Comment:
The initial value *is* defined in the spec, in the property definition table. It is `none`, which should be represented as a string `'none'` since returning `None` in `validation.py` indicates a parse error.